docs/source/running.md

```diff
@@ -156,3 +156,12 @@ Example of the command line (with optional flag to select IB device mlx5_0 port
 $ mpirun -np 2 -env UCX_NET_DEVICES=mlx5_0:1 ./executable
 ```
 
+## Running in Docker containers
+UCX can run in a container, but requires slight adjustments:
+
+* Some transports may be unsupported, depending on the runtime configuration.
+To see the available transports use `ucx_info -d`.
+
+* In order to enable shared memory transport, ptrace capability is required (`--cap-add CAP_SYS_PTRACE`). Shared memory size can be adjusted by `--shm-size=""`.
+
+* To enable RDMA, the corresponding devices must be allowed, e.g `--device=/dev/infiniband/rdma_cm --device=/dev/infiniband/uverbsX`.
```

Comment:
1. is it possible to specify something like `/dev/infiniband/*`?
2. if not, can you clarify what X stands for?
(maybe trivial but we want to avoid user questions)